### DISCUSSION TITLE
Call for Documentation Topics

### DISCUSSION THREAD

**Discussion by Tpada13** [XDC-Community/XDC-Community.github.io#16] [General]
XDC Foundation has sponsored $25k towards improving XDC Network decentralized documentation / dev portal, and we want to hear from the community on the most immediate needs.  If there is documentation, code snippets, how-to, tutorials, or something else, please list in the discussion and we can work through the top priorities.  Please post requests to help support your development journey.

**Comment by WalterBlueu** [comment]
I would love a detailed tutorial that walks a reader through how to create and deploy an XRC20 token on the XDC Network.

**Comment by alexjorgef** [comment]
Since there is a tutorial for using the network with JavaScript, love to see the documentation on how to use TypeScript as well.

**Reply by Tpada13** [reply]
Thank you for the request.  We will be turning this into a bounty.

**Reply by alexjorgef** [reply]
I can help with that for sure. The example code shouldn't use any web framework.

**Reply by alexjorgef** [reply]
Additionally, this could be a good opportunity to:
* extend the already existing tutorials on hardhat from JavaScript to TypeScript and;
  * [how-to/contract-hardhat.md](https://github.com/XDC-Community/docs/blob/main/how-to/contract-hardhat.md)
  * [how-to/XRC20/Hardhat/XRC20](https://github.com/XDC-Community/docs/blob/main/how-to/XRC20/Hardhat/XRC20/scripts/deploy.js)
* make a tutorial on how to integrate [Web3Modal v2.x](https://github.com/WalletConnect/web3modal) with both js and ts

**Reply by WalterBlueu** [reply]
@alexjorgef I have created two bounties per your suggestion on extending existing hardhat how-to articles to use TypeScript:

https://gitcoin.co/issue/29353
https://gitcoin.co/issue/29354

**Comment by alexjorgef** [comment]
Could be useful to have tutorials on how to bind the ABI with multiple languages. I see the relevance in Python and Go.

**Comment by xxxxx-xdc** [comment]
First, projects built in Ethereum or Solana are ruined by excessive competition or network conflict.
It would be nice to make XDC stand out and allow many talented developers to move to XDC and build ideas.

The second is the use of SBT tokens.
Currently, domestic developers have developed community SNS connection services called Soullink (eth) and Claylink (kaytn).
Looking at XDC's community talent connected in this way, we can see the powerful power of our community.
I think it would be better to increase user identification by utilizing SBT token and XDC WEB3 DOMAIN at the same time.

![KakaoTalk_20221002_165526453](https://user-images.githubusercontent.com/96195888/193443966-919f2883-bc23-4583-a0bb-63a41c349d70.jpg)

**Comment by xxxxx-xdc** [comment]
How about "X LINK" using SBT?

https://twitter.com/soulinksbt/status/1554461178818338818?s=20&t=ZSvMpVjHwexVqUK4VxrwJg

https://github.com/webplusone/sbt
